wps-cloudformation-template.yaml

```diff
@@ -49,6 +49,24 @@ Parameters:
   geoserverWpsEndpoint:
     Type: String
     Default: 'http://geoserver-wps.aodn.org.au/geoserver/wps'
+  chunkSize:
+    Type: String
+    Default: '1024'
+  workerDownloadAttempts:
+    Type: String
+    Default: '/tmp'
```

Comment:
defaults of workerDownloadAttempts and workerDownloadDirectory are the wrong way round

Reply:
Good Find :+1: